analysis.yml

```diff
@@ -21,12 +21,14 @@ study:
                     --repo=/dev/shm/spack-$(SPEC) \
                     --from-commit="" \
                     --to-commit="" \
-                    --spec=$(SPEC) \
+                    --spec=$(SPEC)
```

Comment:
You probably want to quote this to support the full spec syntax.  They can have spaces in them, e.g. `"hdf5 target=skylake"`

Minor but I thought that the pythonic way to run things these days was to have a top-level `drift` directory and have a `__main__.py` in it, then `pyhton3 -m drift` etc.

I don't actually care for this project but FYI.